main.go

```diff
@@ -232,6 +239,9 @@ func startDaemon(c *cli.Context) error {
 		provisioner,
 		serverVersion.GitVersion,
 		pvController.LeaderElection(false),
+		pvController.FailedProvisionThreshold(0),
+		pvController.FailedDeleteThreshold(0),
```

Comment:
The default values for these should be 15. 0 means infinite retrying.

Consider adding a command-line argument to control these defaults as well.

Reply:
@FranciscoMSM Thanks for your feedback.

The number of provisioning and deletion retry thresholds has also been changed to be controlled by user options like (--provisioning-retry-count, --deletion-retry-count).